dask_ml/datasets.py

```diff
@@ -64,12 +64,14 @@ def make_counts(
     rng = dask_ml.utils.check_random_state(random_state)
 
     X = rng.normal(0, 1, size=(n_samples, n_features), chunks=(chunks, n_features))
-    informative_idx = rng.choice(n_features, n_informative, chunks=n_informative)
+    informative_idx = rng.choice(
+        n_features, n_informative, chunks=n_informative, replace=False
+    )
     beta = (rng.random(n_features, chunks=n_features) - 1) * scale
 
     informative_idx, beta = dask.compute(informative_idx, beta)
 
-    z0 = X[:, informative_idx].dot(beta[informative_idx])
+    z0 = X[:, informative_idx].dot(beta[informative_idx]) + 0.5
```

Comment:
Why are changes to this file required?

Reply:
This adds an explicit intercept term, as the `X` array does not have a constant term for the intercept. I am happy to remove it, however, as it does not change the results.